src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -227,9 +227,10 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal with CassandraReco
         bs.setString("event_manifest", m.eventManifest)
         bs.setBytes("event", m.serialized)
         // for backwards compatibility
-        bs.setToNull("message")
+        bs.unset("message")
```

Comment:
made another commit which has compatibility fix, unset is supported only after protocol version 4